src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
```

Comment:
The @Data annotation includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. Using all of them together may violate the Single Responsibility Principle by combining multiple responsibilities in one class.